docs/elements/17-instagram.md

```diff
@@ -0,0 +1,45 @@
+# Instagram Element
```

Comment:
What are thoughts on removing the list of CAPI thrift fields and also AMP instagram fields and just keeping the links?

I am still slightly concerned that we are creating documentation that is going to get out of date and just duplicates canonical sources.

Reply:
that was just so I didn't have to have too many windows open, will fix
![](https://media.giphy.com/media/Au9yFbsczWaHe/giphy.gif)